src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -128,7 +152,7 @@ class HoldRequest extends React.Component {
 
             <input type="hidden" name="pickupLocation" value={location.code} />
 
-            <button type="submit" className="large">
+            <button type="submit" className="large" onClick={(e) => this.submitRequest(e, itemId)}>
```

Comment:
We might need a fall back for this button in case no JS. Could be a separated PR though.